.changeset/nice-monkeys-rest.md

```diff
@@ -0,0 +1,5 @@
+---
+"@meilisearch/instant-meilisearch": patch
+---
+
+Migrate the current repository to a turbo architecture
```

Comment:
while this has nothing to do with instant-meilisearch, I sill need to add it as a patch if I want it to be released on npm